manifests/prepare.pp

```diff
@@ -51,14 +53,87 @@
     }
   }
 
-  # TODO: manage server.cfg contents
-  if $::mcollective_configured {
-    file { '/etc/puppetlabs/mcollective':
-      ensure => directory,
+  # manage client.cfg and server.cfg contents
+  file { '/etc/puppetlabs/mcollective':
+    ensure => directory,
+  }
+  # The mco_*_config facts will return the location of mcollective config (or nil), prefering PE over FOSS.
+  if $::mco_server_config {
+    $mco_server = '/etc/puppetlabs/mcollective/server.cfg'
+    file { $mco_server:
+      ensure  => file,
+      source  => $::mco_server_config,
+    }
+
+    if $::mco_server_settings {
+      $mco_server_libdir = $::mco_server_settings['libdir']
+      if $mco_server_libdir {
+        ini_setting { 'server/libdir':
+          section => '',
+          setting => 'libdir',
+          path    => $mco_server,
+          value   => "${mco_server_libdir}:/opt/puppetlabs/mcollective/plugins",
```

Comment:
I'll update this to only append if `libdir` doesn't already contain the new path. Same with `plugin.yaml`.